src/status_im/contexts/profile/profiles/view.cljs

```diff
@@ -256,6 +256,12 @@
             (rf/dispatch
              [:keycard/connect
               {:key-uid key-uid
+               :on-error
+               (fn [error]
+                 (rf/dispatch [:keycard/on-application-info-error
+                               (if (= error :keycard/error.keycard-empty)
+                                 :keycard/error.keycard-wrong-profile
```

Comment:
could you elaborate on this condition pls

Reply:
hi @flexsurfer, Thank you very much for reviewing the PR.

As reported in [this issue](https://github.com/status-im/status-mobile/issues/22025), we need to show same different keycard error even when logging in with empty keycard.